include/zephyr/dt-bindings/pinctrl/numaker_m46x_pinctrl.h

```diff
@@ -0,0 +1,2154 @@
+/*
```

Comment:
by convention of all other files, use `numaker-m46x-pinctrl.h` file name

Reply:
Yes, it's better to match convention.